app/views/unattended/provisioning_templates/snippet/first_boot_setup_script.erb

```diff
@@ -0,0 +1,9 @@
+<%#
+kind: snippet
+name: first_boot_setup_script
+model: ProvisioningTemplate
+snippet: true
+description: |
+    First boot setup script
+-%>
+<%= indent(2, skip1: true) { snippet('built', :variables => { :endpoint => 'built', :method => 'POST', :body_file => '/root/install.post.log' }) } -%>
```

Comment:
Do we need this as a separate snippet? IMHO this could be part of first_boot_setup.erb.

Reply:
I have merged it into the main template. But it might be cleaner to create separate sniper for both systemd and non systemd as you say.